app/i18n/ja/admin.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+return array(
+	'auth' => array(
+		'allow_anonymous' => '標準のユーザーの記事が匿名のユーザーでも読めるようにします。\' (%s)',	// TODO - Translation
```

Comment:
Not super important I suppose, but these don't look very todo. ^_^

Reply:
Thank you for pointing out the issue!
I remove ` // TODO - Translation `